linkingFuncs.py

```diff
@@ -33,11 +33,43 @@ def get_profile(username):
     cur = cnx.cursor(buffered=True)
     id = get_id(username, cur)
     #get transactions
-    cur.execute("SELECT * FROM Transactions WHERE UserId='%d'" % (id))
+    cur.execute("SELECT * FROM Transactions WHERE UserId='%d' ORDER BY ticker" % (id))
```

Comment:
It may make sense to order by the date of the transaction as opposed to alphabetical which I believe ordering by ticker will do. I'm not sure if we track the date of the transaction, but I believe it's a pretty easy thing to track: https://www.programiz.com/python-programming/datetime/current-datetime.

Reply:
LGTM

Reply:
I will look into it